### DISCUSSION TITLE
More than 16 lists

### DISCUSSION THREAD

**Discussion by DavidWells** [community/community#9672] [Other Feature Feedback, Questions, & Ideas] [answered]
I have over 4.2k starred repos of all different genres.

It would be awesome to have as many lists as I need to organize the libraries.

16 is much too limiting for how I'd like to group repos. 

Thanks for adding this awesome feature. It will surely help with code discoverability. ❤️

**Comment by Katsute** [comment, marked as answer]
https://github.com/github/feedback/discussions/8633
